### ISSUE TITLE
RetinaJS not work when using jQuery.load() to load only DIV content

### ISSUE THREAD

**Issue by kench928** [imulus/retinajs#199]
I have an issue when using jQuery.load() to load only DIV content.
Here are what I am doing:

$('body').delegate('.btn a', 'click', function(){
	$('.temp').load('abc.html #div', function(response, status, xhr) {
		if(status == "success") {
			// do something...
		}
	});
});

After click a button, it load the '#div' inside 'abc.html' and append to body.
The images inside 'abc.html #div' not scanned by RetinaJS because it is loaded after RetinaJS run.
Is there any method to recall the RetinaJS to scan again?

Thanks

**Comment by ludydoo** [comment]
+1

**Comment by withlovee** [comment]
+1

**Comment by orenonou** [comment]
A hack solution would be to actually name the anonymous function and manually call it every time you need it...

Start of the code: function RetinaJS(){function a(){}function b(a){r....
End of the code: ...path);var c=this;b()},a.isRetina()&&a.init(e)};

Then simple call RetinaJS() when you need it.

**Comment by jgnewman** [comment]
Hi everyone! We released 2.0 today and it now allows you to call the script again whenever you need to. Hopefully that helps!

**Comment by miela** [comment]
Hi! But in 2.0 is this issue too. I just checked. When I load page via jQuery.load() retinaJS doesn't work. Any tip?

I tried add JS file in main HTML and in loaded by jQuery (and both) - still nothing.

**Comment by jgnewman** [comment]
@miela Yep, that's expected but there's now a way around it. There are a couple things to keep in mind. First of all, 2.0.0's API is totally different than 1.x so you'll want to make sure you've structured your code appropriately. Secondly, the script still runs automatically on page load but as of 2.0.0, it exposes a global function called `retinajs` on the `window` object so that _after_ you load your page over ajax, you can just call `retinajs()` to run the script again. So that would look something like this:

```javascript
$.get('my_page.html', function (data) {
  $('#container').html(data);
  retinajs();
});
```

**Comment by miela** [comment]
@jgnewman Hey, thanks. It helps.
